src/interactive/repl.ts

```diff
@@ -149,10 +149,10 @@ async function startREPL(preserveFocus: boolean, showTerminal: boolean = true) {
     }
 }
 
-function juliaConnector(pipename, start = false) {
+function juliaConnector(pipename: string, start = false) {
     const connect = `VSCodeServer.serve("${pipename}"; is_dev = "DEBUG_MODE=true" in Base.ARGS, crashreporting_pipename = "${telemetry.getCrashReportingPipename()}");nothing # re-establishing connection with VSCode`
```

Comment:
I think the two strings here also need the `raw` treatment. Part two of an example string on windows:

```
VSCodeServer.serve("\\.\pipe\vsc-jl-repl-be33bd08-7bf2-48bd-a908-e9f263a6845e"; is_dev = "DEBUG_MODE=true" in Base.ARGS, crashreporting_pipename = "\\.\pipe\vsc-jl-cr-a6536eef-67ab-4b9d-a4e2-a35b5c36610f");nothing # re-establishing connection with VSCode
```

Reply:
Ah, good catch.